h5bp/security/x-frame-options.conf

```diff
@@ -27,6 +26,9 @@
 # more than just clickjacking attacks.
 # https://cure53.de/xfo-clickjacking.pdf.
 #
+# Note: The Content-Security-Policy HTTP header has a frame-ancestors directive
+# which obsoletes this header for supporting browsers.
```

Comment:
Not in the issue #275, but it could be cool to add this info copied from https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options

Reply:
Please don't, as CSP is not enabled per default, and since the support is a bit better.

Reply:
@LeoColomb sorry for the delay. 

We could see at https://github.com/h5bp/server-configs-nginx/blob/d249bf2f2bfa51bb6018e73094da6c721ecb0ad1/nginx.conf#L112,L116 than seems to be enabled by defaut ?

Reply:
Note, I've past this assertion from https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options . You want remove it ?

Reply:
> than seems to be enabled by defaut

Indeed, sorry about that.